Library/ApplePayCapable.swift

```diff
@@ -0,0 +1,111 @@
+import Foundation
+import KsApi
+import PassKit
+
+public protocol ApplePayCapableType {
```

Comment:
Just wondering about the name seeing as it does a little more than just tell us whether the device is capable of Apple Pay. Anyway, no super strong feelings about it, but just `ApplePayType` would probably be fine too 👍

Reply:
Yeah the term I was going for here is _capabilities_ to encompass all Apple-Pay-related capabilities. But `ApplePayCapabilitiesType` didn't quite have the same ring to it 😅. `ApplePayType` and then `ApplePay` seems a little vague to me.

Reply:
I personally like `ApplePayCababilities`, because it's more general. For me, `ApplePayCapable` refers to something with a boolean type (function, var...).